src/pymmcore_widgets/hcs/_main_wizard_widget.py

```diff
@@ -0,0 +1,510 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+from typing import List, Optional, Union, cast
+
+from platformdirs import user_data_dir
+from pymmcore_plus import CMMCorePlus
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtWidgets import (
+    QFileDialog,
+    QSizePolicy,
+    QSpacerItem,
+    QVBoxLayout,
+    QWidget,
+    QWizard,
+    QWizardPage,
+)
+from useq import GridRowsColumns, MDASequence, Position, RandomPoints
+
+from ._calibration_widget import CalibrationData, PlateCalibrationWidget
+from ._fov_widget import Center, FOVSelectorWidget
+from ._graphics_items import Well  # noqa: TCH001
+from ._plate_model import DEFAULT_PLATE_DB_PATH, Plate, load_database, save_database
+from ._plate_widget import PlateInfo, PlateSelectorWidget
+from ._pydantic_model import FrozenModel
+from ._util import apply_rotation_matrix, get_well_center, nearest_neighbor
+
+EXPANDING = (QSizePolicy.Policy.Expanding, QSizePolicy.Policy.Expanding)
+TOP_X: float = -10000000000
+TOP_Y: float = 10000000000
+# Path to the user data directory to store the plate database
+USER_DATA_DIR = Path(user_data_dir(appname="pymmcore_widgets"))
```

Comment:
we should expose this via pymmcore_plus so you don't need to use platformdirs directly here